asreview/webapp/src/ProjectComponents/ProjectPage.js

```diff
@@ -85,13 +104,62 @@ const ProjectPage = (props) => {
     }
   );
 
+  const isSimulating = () => {
```

Comment:
Where is `data` defined? I don't fully understand how this works.

Reply:
It is the data fetched from the `info` API above. Basically, it uses `mode`, `projectInitReady`, and `reviewFinished` in the returned project info to judge if the simulation is finished.